libraries/AP_Motors/AP_MotorsTailsitter.cpp

```diff
@@ -101,10 +101,55 @@ void AP_MotorsTailsitter::output_to_motors()
 // calculate outputs to the motors
 void AP_MotorsTailsitter::output_armed_stabilizing()
 {
-    _aileron = -_yaw_in;
-    _elevator = _pitch_in;
-    _rudder = _roll_in;
-    _throttle = get_throttle();
+    float   roll_thrust;                // roll thrust input value, +/- 1.0
+    float   pitch_thrust;               // pitch thrust input value, +/- 1.0
+    float   yaw_thrust;                 // yaw thrust input value, +/- 1.0
+    float   throttle_thrust;            // throttle thrust input value, 0.0 - 1.0
+    float   rpy_thrust;                 // the minimum throttle setting that will not limit the roll pitch and yaw output
+    float   thr_adj;                    // the difference between the pilot's desired throttle and throttle_thrust_best_rpy
+
+    // apply voltage and air pressure compensation
+    // roll, pitch, yaw should only be scaled by air density
+    roll_thrust = _roll_in * get_compensation_gain();
+    pitch_thrust = _pitch_in * get_compensation_gain();
+    yaw_thrust = _yaw_in * get_compensation_gain();
+    throttle_thrust = get_throttle() * get_compensation_gain();
```

Comment:
In my testing, get_compensation_gain() always returns 1.0. Is this functional for quadplanes?